### PR TITLE
Finished Test

### PR DESCRIPTION
**Pull request by Lee1312**

### CODE REVIEW

File: src/HospitalLibrary/AnnualLeaves/Interfaces/IAnnualLeaveValidator.cs [hunk 1]
```diff
@@ -5,5 +5,6 @@ namespace HospitalLibrary.AnnualLeaves.Interfaces
     public interface IAnnualLeaveValidator
     {
         void Validate(AnnualLeave annualLeave);
+        void Cancel_Validation(AnnualLeave leave, int doctorId);
```
Comment (Ognjenas) [inline src/HospitalLibrary/AnnualLeaves/Interfaces/IAnnualLeaveValidator.cs:8]: Change to CancelValidation

**Review by Ognjenas — COMMENTED** [review]
Change method name

**Review by Lee1312 — COMMENTED** [review]
Renamed

**Review by Lee1312 — COMMENTED** [review]
Renamed all

**Review by Ognjenas — APPROVED** [review]

**Pull request merged**